src/condor_startd.V6/VolumeManager.cpp

```diff
@@ -304,7 +304,7 @@ VolumeManager::CreateLoopback(const std::string &filename, uint64_t size_kb, Con
 
         // TODO: Would be friendlier to unmount everything and resize instead of failing
     if (backing_stat.st_size < static_cast<off_t>(size_kb*1.02)*1024) {
-        err.pushf("VolumeManager", 6, "Pre-existing loopback file is too small (is %ld, asking for %ld\n)", backing_stat.st_size, (size_kb * 1.02) / 1024);
+        err.pushf("VolumeManager", 6, "Pre-existing loopback file is too small (is %g, asking for %g\n)", (double) backing_stat.st_size, (size_kb * 1.02) * 1024);
```

Comment:
Why was this change made?  It seems like a good way to get bad data.

Reply:
These numbers can both be large, as there are in bytes, one is a double, the other a uint64_t, but printing one in scientific notation and the other as an integer makes it hard for a user to compare scales.

Reply:
It would probably be more correct to store the truncation being done by the previous static cast and print them both as integers, but hopefully it will never matter.